app/views/coronavirus_landing_page/components/shared/_related_topic_section.html.erb

```diff
@@ -0,0 +1,32 @@
+<section class="covid__topic-wrapper">
+  <%= render "govuk_publishing_components/components/heading", {
+    text: related_topic_section["header"],
+    padding: true,
+    border_top: 2,
+    margin_bottom: 6
+  } %>
+
+  <ul class="govuk-list">
+    <% related_topic_section["links"].each do | link | %>
+      <li class="covid__topic-list-item">
+        <%
+          data = if tracking
```

Comment:
You seem to have an unnecessary conditional here (that perhaps errors, I assume tracking isn't defined?)

Reply:
Sorry - will fix